test/specs/commonTests.js

```diff
@@ -858,31 +858,13 @@ export const implementsImageProp = (Component, options = {}) => {
  * @param {Object} [options.requiredProps={}] Props required to render the component.
  */
 export const implementsTextAlignProp = (Component, options = {}) => {
-  const { requiredProps = {} } = options
   const { assertRequired } = commonTestHelpers('implementsTextAlignProp', Component)
 
   describe('aligned (common)', () => {
     assertRequired(Component, 'a `Component`')
 
     _noDefaultClassNameFromProp(Component, 'textAlign', options)
     _noClassNameFromBoolProps(Component, 'textAlign', options)
-
-    _.each(Component._meta.props.aligned, (propVal) => {
```

Comment:
I'm think we can do there something better :thinking: We can use this pattern with other common tests that relies on `meta`. Thoughts?

```js
export const implementsTextAlignProp = (Component, alignments = SUI.TEXT_ALIGNMENTS,  options = {}) => {
// ...
  _.each(alignments, (propVal) => {
  // ...
  })
}
```